package.json

```diff
@@ -13,17 +13,19 @@
     "dist"
   ],
   "scripts": {
-    "test": "mocha --compilers js:./babel-register --inline-diffs --check-leaks --ui tdd --reporter dot test",
-    "build": "babel --modules commonStrict --source-maps-inline --out-dir dist src",
+    "test": "mocha --compilers js:babel-core/register --inline-diffs --check-leaks --ui tdd --reporter dot test",
+    "build": "babel --source-maps-inline --out-dir dist src",
     "prepublish": "rm -rf dist && npm update && npm run build"
   },
   "dependencies": {
-    "shift-spec": "^2015.1.3"
+    "shift-spec": "2015.2.0"
```

Comment:
Admittedly I don't know much about how npm does versioning, but it certainly seemed to: "^2015.2.0" did not appear to force an update from 2015.1.3. This may have been an artifact of my setup, but "^2015.2.0" didn't build, and "2015.2.0" did.